Kudu.Services.Web/KubeMiddleware.cs

```diff
@@ -57,13 +62,24 @@ public async Task Invoke(HttpContext context, IEnvironment environment, IServerC
                 homeDir = "/home/apps/";
                 siteRepoDir = "/site/repository";
             }
+            // TODO test the Items.
+            Console.WriteLine("*********************** check the context.Items");
+            foreach (var item in context.Items)
+            {
+                Console.WriteLine($"****** context.item: {item.Key}");
+            }
+
+            Console.WriteLine("*********************** check the context.Items");
```

Comment:
remove?

Reply:
Yes. It is just debugging for the testing. I'm testing with k4app, however, it doesn't work (It fails with patching) I'm investigating it for now.

Reply:
This issue. I create a ticket to track. I'll find the root cause and fit it. https://github.com/microsoft/k4apps/issues/496

Reply:
Removed, Some part, @sanchitmehta  want it, so that we include it with plain-old `if def` definition. so that it is not included for production.